tpdatasrc/tpgamefiles/scr/tpModifiers/bonus_spell.py

```diff
@@ -0,0 +1,68 @@
+from templeplus.pymod import PythonModifier
+from toee import *
+import tpdp
+
+print "Registering Bonus Spell Slot Feats"
+
+def setSpellLevelArg(attachee, args, evt_obj):
+    spellLevel = args.get_param(0)
+    args.set_arg(2, spellLevel)
+    return 0
+
+def applyExtraSpell(attachee, args, evt_obj):
+    classEnum = args.get_arg(1)
+    spellLevel = args.get_arg(2)
+    #I am unsure why evt_obj.caster_class is not working, can't get a print of it either
+    #print "evt_obj.caster_class: {}".format(evt_obj.caster_class)
+    #Disabeld for now due to this
+    #if not evt_obj.caster_class == classEnum:
```

Comment:
I've added get_caster_class() method to the event object; I think the issue was that it was directly accessing an enum field which makes it typed in python, and this causes the issue

Reply:
Thanks that fixed the issue!